app/livechat/server/publications/visitorPageVisited.js

```diff
@@ -30,6 +30,24 @@ Meteor.publish('livechat:visitorPageVisited', function({ rid: roomId }) {
 
 		self.ready();
 
+		self.onStop(function() {
+			handle.stop();
+		});
+	} else if (roomId === '' && token !== '') {
```

Comment:
Why this else?

Reply:
All this to add navigation history without room